### DISCUSSION TITLE
Evaluating an integral of the solution

### DISCUSSION THREAD

**Discussion by cjham** [firedrakeproject/firedrake#3688] [Firedrake support] [answered]
Hello,
This is probably a very basic question but I can't see the answer. I'm sorry if I have missed it.
I would like to evaluate an integral of a function of my solution over the domain and print it. How can I do this?
Many thanks,

**Comment by colinjcotter** [comment, marked as answer]
if your function is called u,

a = assemble(u*dx)

**Comment by cjham** [comment]
Great, thank you!
